app.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const routing = require('./routes/routing');
+const buzzwords = require('./routes/buzz-words');
+const express = require('express');
+const app = express();
+
+app.use('/', routing);
+app.use('/buzzwords', buzzwords);
+
+const server = app.listen(3000, () => {
+  const host = server.address().address;
```

Comment:
Your port and host variables should be declared at the top of your files, below the packages and files you're requiring. You should be using process.env.PORT.